packages/fluentui/projects-test/package.json

```diff
@@ -7,14 +7,26 @@
     "@fluentui/react-northstar": "^0.51.0"
   },
   "devDependencies": {
+    "@lerna/project": "^3.15.0",
```

Comment:
Not super important but if there are any of these deps which are no longer used in `scripts`, would you mind removing them there?

Reply:
Actually `@lerna/project` & `@lerna/package-graph` are new dependencies. They are used to populate required dependencies. I've checked, only `tmp` can be removed from `scripts/package.json`, that's done